README.md

```diff
@@ -1 +1,2 @@
 #Dani's first comment.
+#Jedeo's first Comment.
```

Comment:
OHHHHHHHH, it's right hurrrrrr!
Good job, I really like this one you added here. Why didn't I think of that? lol

Reply:
alright I see you